src/test/testType.ts

```diff
@@ -34,6 +34,9 @@ export class TestTypeImpl {
   constructor(fixtures: (FixturesWithLocation | DeclaredFixtures)[]) {
     this.fixtures = fixtures;
 
+    if (process.env.JEST_WORKER_ID)
```

Comment:
Let's maybe move this to the test() or describe() function so that I can require @playwright/test and use `chromium.launch` in jest? Seems rude to throw during require.

Reply:
I personally think we should 100% prevent that from doing. If they want to use chromium.launch in Jest, then they should use the normal playwright* packages.